crates/indexer/src/accountsdb/accounts/token.rs

```diff
@@ -1,95 +1,14 @@
 use chrono::offset::Local;
 use indexer_core::{
-    db::{
-        insert_into,
-        models::{TokenAccount as TokenAccountModel, TokenTransfer},
-        select,
-        tables::{metadatas, token_accounts, token_transfers},
-        update,
-    },
+    db::{insert_into, models::TokenAccount as TokenAccountModel, tables::token_accounts},
     prelude::*,
-    pubkeys::{kin, sol},
 };
 use spl_token::state::Account as TokenAccount;
 
 use super::Client;
 use crate::prelude::*;
 
-pub async fn process_token_transfer(
-    client: &Client,
-    key: Pubkey,
-    token_account: TokenAccount,
-) -> Result<()> {
-    let accts = client
-        .db()
-        .run(move |db| {
-            token_accounts::table
-                .filter(token_accounts::address.eq(&key.to_string()))
-                .load::<TokenAccountModel>(db)
-        })
-        .await
-        .context("Failed to check token_accounts!")?;
-    let acct = accts
-        .get(0)
-        .context("Could not find existing token account!")?;
-    if token_account.owner.to_string() != acct.owner_address {
-        let row = TokenTransfer {
-            owner_from: Owned(acct.owner_address.to_string()),
-            owner_to: Owned(token_account.owner.to_string()),
-            mint_address: Owned(token_account.mint.to_string()),
-            transferred_at: Local::now().naive_utc(),
-        };
-        client
-            .db()
-            .run(move |db| {
-                insert_into(token_transfers::table)
-                    .values(&row)
-                    .on_conflict_do_nothing()
-                    .execute(db)
-            })
-            .await
-            .context("failed to insert into token_transfers table")?;
-        client
-            .db()
-            .run(move |db| {
-                update(token_accounts::table)
-                    .filter(token_accounts::address.eq(&key.to_string()))
-                    .set(token_accounts::owner_address.eq(&token_account.owner.to_string()))
-                    .execute(db)
-            })
-            .await
-            .context("failed to update token account!")?;
-    }
-
-    Ok(())
-}
 pub async fn process(client: &Client, key: Pubkey, token_account: TokenAccount) -> Result<()> {
-    if token_account.mint == sol() {
-        return Ok(());
-    }
-    if token_account.mint == kin() {
-        return Ok(());
-    }
```

Comment:
I think we could either keep this, or if we remove it we should get rid of the definitions for `sol` and `kin` in pubkeys.rs